charts/datastore/README.md

```diff
@@ -39,6 +39,7 @@ The data-on-prem solution runs with one of 3 modes (datastore.serverMode):
 | Parameter | Description |
 | ------ | ------ |
 | datastore.serverMode | Can have only 3 values: **AUTOTLS**, **TLS**, **PLAIN**|
+| datastore.ingressPort | Ingress external port (default=80). See more ingress guide lines below this table |
```

Comment:
In the bottom line it isn't ingress specific port, but just additional one, so the **name** is a bit misleading.
maybe a user would want an additional port but not for ingress (maybe haproxy? nginx?)

Reply:
maybe external port ?

Reply:
Additional-port is enough IMO, as in the readme 80 and 433 are external as well.
and they are exactly external, depends on the service-type

Reply:
they are not*